openocd_cypress/openocd_cypress.mk

```diff
@@ -4,7 +4,7 @@
 #
 ################################################################################
 
-OPENOCD_CYPRESS_VERSION = release-v3.0.0
+OPENOCD_CYPRESS_VERSION = release-v4.1.0
```

Comment:
@erauer Be sure to mention "openocd_cypress v4.1.0" in the commit message - it's not just an x86-64 build fix.